engine/xml-config/src/test/resources/JaxbJobWriterTest-testWriteCsvTemplate.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<job xmlns="http://eobjects.org/analyzerbeans/job/1.0">
+    <job-metadata>
+        <job-version>2.0</job-version>
+        <author>John Doe</author>
+        <created-date>2010-11-12Z</created-date>
+    </job-metadata>
+    <source>
+        <data-context ref="my db"/>
+        <columns>
+            <column id="col_firstname" path="EMPLOYEES.FIRSTNAME" type="VARCHAR"/>
+            <column id="col_lastname" path="EMPLOYEES.LASTNAME" type="VARCHAR"/>
+            <column id="col_email" path="EMPLOYEES.EMAIL" type="VARCHAR"/>
+        </columns>
+        <variables>
+            <variable id="psp.output.path.final" value="/Users/claudiap/Documents/OutgoingHotFolder"/>
+            <variable id="hotfolder.input.filename" value="myFile"/>
+            <variable id="datacleaner.run.timestamp" value="1482244133378"/>
+        </variables>
+    </source>
+    <transformation/>
+    <analysis>
+        <analyzer>
+            <descriptor ref="Create CSV file"/>
+            <metadata-properties>
+                <property name="Template value.File">${psp.output.path.final/hotfolder.input.filename/datacleaner.run.timestamp-samples.csv}</property>
```

Comment:
This sample is wrong. Properties should always be individually wrapped. This value should be: `${psp.output.path.final}/${hotfolder.input.filename}/${datacleaner.run.timestamp}-samples.csv`